pkg/yurthub/certificate/token/token.go

```diff
@@ -274,6 +277,9 @@ func (ycm *yurtHubCertManager) getPkiDir() string {
 
 // getBootstrapConfFile returns the path of yurthub bootstrap conf file
 func (ycm *yurtHubCertManager) getBootstrapConfFile() string {
+	if len(ycm.bootstrapFile) != 0 {
+		return ycm.bootstrapFile
+	}
```

Comment:
we need to check all codes that invoke `getBootstrapConfFile()` function can work or not if Yurthub restart and bootstrap file doesn't exist. 

The following places can not work:
1. verifyServerAddrOrCleanup()
2. prepareConfigAndCaFile()
3. generateCertClientFn()